docker-compose/docker-compose.yaml

```diff
@@ -0,0 +1,59 @@
+---
+version: "3.7"
+services:
+  mariadb:
+    image: mariadb:10.5
+    restart: unless-stopped
+    environment:
+      MARIADB_ROOT_PASSWORD: password
+      MARIADB_DATABASE: buildbot
+      MARIADB_USER: buildmaster
+      MARIADB_PASSWORD: password
```

Comment:
can you please add a healthcheck, something like:
```yaml
    healthcheck:
      test: ['CMD', "mariadb-admin", "--password=${ENV_MARIADB_ROOT_PASSWORD}", "--protocol", "tcp", "ping"]
```

See: https://gitlab.com/mariadb/downloads-mariadb-python3/-/blob/master/docker/docker-compose.yml

Reply:
The link is broken

Reply:
You have no access rights I guess.